ondemand/install.sh

```diff
@@ -7,6 +7,183 @@ log_info() {
+    --with-http_xslt_module \
+    --with-http_image_filter_module \
+    --with-http_sub_module \
+    --with-http_dav_module \
+    --with-http_flv_module \
+    --with-http_mp4_module \
+    --with-http_gunzip_module \
+    --with-http_gzip_static_module \
+    --with-http_random_index_module \
+    --with-http_secure_link_module \
+    --with-http_degradation_module \
+    --with-http_stub_status_module \
+    --with-mail \
+    --with-mail_ssl_module \
+    --with-pcre \
+    --with-pcre-jit \
+    --add-module=../src/nginx_module \
+    --with-cc-opt="$NGINX_CCOPTS" \
+    --with-ld-opt="$LDOPTS" \
+    --with-debug
+
+  make -j$(nproc) && make install INSTALLDIRS=vendor
+  mkdir -p  $NGINX_DATADIR/html
+  mkdir -p  $NGINX_CONFDIR
+  mkdir -p  $NGINX_HOME
+  mkdir -p  $NGINX_HOME_TMP
+  mkdir -p  $NGINX_LOGDIR
+
+  set -x
+  cd "$BUILD_DIR/passenger-$PASS_VERSION"
+  which ruby
+	rake fakeroot \
+	    NATIVE_PACKAGING_METHOD=rpm \
+	    FS_PREFIX=$PREFIX \
+	    FS_BINDIR=$PREFIX/bin \
+	    FS_SBINDIR=$PREFIX/sbin \
+	    FS_DATADIR=$PREFIX/usr/share \
+	    FS_LIBDIR=$PREFIX/lib64 \
+	    FS_DOCDIR=$PREFIX/usr/share/doc \
+	    RUBY=$(which ruby) \
+	    RUBYLIBDIR=$RUBY_LIBDIR \
+	    RUBYARCHDIR=$RUBY_LIBDIR \
+	    APACHE2_MODULE_PATH=$PREFIX/usr/lib/apache2/modules/mod_passenger.so \
+			OPTIMIZE=yes \
+			CACHING=false \
+			EXTRA_CFLAGS="$PASSENGER_CCOPTS" \
+			EXTRA_CXXFLAGS="$PASSENGER_CCOPTS"
+
+  cp -a $BUILD_DIR/passenger-$PASS_VERSION/pkg/fakeroot/* /
+  cd $BUILD_DIR/passenger-$PASS_VERSION
+  ./dev/install_scripts_bootstrap_code.rb --ruby $RUBY_LIBDIR \
+	  $PREFIX/bin/passenger* \
+	  $PREFIX/sbin/passenger* \
+	  `find $PREFIX -name rack_handler.rb`
+
+  ./dev/install_scripts_bootstrap_code.rb --nginx-module-config $PREFIX/bin $PREFIX/usr/share/passenger/ngx_http_passenger_module/config
+  chmod +x $PREFIX/usr/share/passenger/helper-scripts/wsgi-loader.py
+}
+
+install_os_deps() {
+  dnf -y update && \
+    dnf install -y dnf-utils epel-release && \
+    dnf config-manager --set-enabled powertools && \
+    dnf -y module enable nodejs:12 ruby:2.7 && \
+    dnf install -y \
+        file lsof sudo gcc gcc-c++ git \
+        patch lua-posix rsync ruby ruby-devel python2 python3 \
+        nodejs sqlite sqlite-devel nmap-ncat httpd httpd-devel mod_ssl \
+        libcurl-devel autoconf openssl-devel jansson-devel libxml2-devel \
+        libxslt-devel gd-devel
+  gem install rake dotenv
+}
+
+build_ood_src() {
+  cd $BUILD_DIR
+
+  git clone https://github.com/cisco/cjose
+  cd cjose
+  git checkout 0.6.1
+  ./configure
+  make && make install
+
+  # so mod_auth_openidc can find cjose
+  export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig
+
+  cd $BUILD_DIR
+  git clone https://github.com/zmartzone/mod_auth_openidc.git
+  cd mod_auth_openidc
+  git checkout v2.4.9.4
+  sh autogen.sh
+  ./configure
+  make && make install
+
+  cd $BUILD_DIR
+  git clone https://github.com/OSC/ondemand.git
+  cd ondemand
```

Comment:
I'm now remembering that I probably need to set this back to a tag instead the mainline.

Reply:
Yeah good idea. Should be an easy fix.